content-src/components/DiscoveryStreamComponents/Hero/Hero.jsx

```diff
@@ -1,7 +1,27 @@
+import {actionCreators as ac} from "common/Actions.jsm";
 import {DSCard} from "../DSCard/DSCard.jsx";
 import React from "react";
 
 export class Hero extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.onLinkClick = this.onLinkClick.bind(this);
+  }
+
+  onLinkClick(event) {
+    this.props.dispatch(ac.UserEvent({
+      event: "CLICK",
+      source: this.props.type.toUpperCase(),
+      action_position: 0,
```

Comment:
I guess hero cards are always going to be in position `0`?

Reply:
yes, at least `[heroRec, ...restCards] = component.data.recommendations` suggests so.